fmc/src/flow/rt_alias.rs

```diff
@@ -11,23 +11,37 @@ Abstract:
     Crypto helper routines
 
 --*/
-
 use crate::flow::dice::{DiceInput, DiceLayer, DiceOutput};
+use crate::flow::pcr::{extend_pcr0, extend_pcr1};
 use crate::fmc_env::FmcEnv;
+use crate::HandOff;
+use caliptra_common::cprintln;
 use caliptra_drivers::CaliptraResult;
 
 #[derive(Default)]
 pub struct RtAliasLayer {}
 
 impl DiceLayer for RtAliasLayer {
     /// Perform derivations for the DICE layer
-    fn derive(env: &FmcEnv, _: &DiceInput) -> CaliptraResult<DiceOutput> {
-        Self::extend_pcrs(env)?;
-        Err(0)
+    fn derive(env: &FmcEnv, _input: &DiceInput) -> CaliptraResult<DiceOutput> {
+        // At this point PCR0 & PCR1 must have the same value. We use the value
+        // of PCR1 as the UDS for deriving the CDI
```

Comment:
PCRs hold measurements, shouldn't we be looking in KV slots for UDS / CDI? Also FMC doesn't have access to UDS.

Reply:
This is a comment from the code I lifted from ROM, which is quite confusing. PCR1 holds measurements but they call it UDS but it should  be the CDI of the previous layer.